src/rmodels.c

```diff
@@ -5039,11 +5064,39 @@ static Model LoadGLTF(const char *fileName)
             }
         }
 
-/*
         // TODO: Load glTF meshes animation data
         // REF: https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.html#skins
         // REF: https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.html#skinned-mesh-attributes
         //----------------------------------------------------------------------------------------------------
+
+        if(data->skins_count > 0) {
+            if(data->skins_count != 1) {
+                TRACELOG(LOG_ERROR, "MODEL: [%s] can only load one skin (armature) per model, but gltf skins_count == %i", fileName, data->skins_count);
+            } else {
+                cgltf_skin skin = data->skins[0];
+                model.bones = LoadGLTFBoneInfo(skin, &model.boneCount);
+                model.bindPose = RL_MALLOC(model.boneCount * sizeof(Transform));
+
+                for(unsigned int i = 0; i < model.boneCount; i++) {
+                    cgltf_node node = *skin.joints[i];
+                    model.bindPose[i].translation.x = node.translation[0];
+                    model.bindPose[i].translation.y = node.translation[1];
+                    model.bindPose[i].translation.z = node.translation[2];
+
+                    model.bindPose[i].rotation.x = node.rotation[0];
+                    model.bindPose[i].rotation.y = node.rotation[1];
+                    model.bindPose[i].rotation.z = node.rotation[2];
+                    model.bindPose[i].rotation.w = node.rotation[3];
+
+                    model.bindPose[i].scale.x = node.scale[0];
+                    model.bindPose[i].scale.y = node.scale[1];
+                    model.bindPose[i].scale.z = node.scale[2];
+                }
+
+                BuildPoseFromParentJoints(model.bones, model.boneCount, model.bindPose);
+            }
+        }
+
         for (unsigned int i = 0, meshIndex = 0; i < data->meshes_count; i++)
         {
             for (unsigned int p = 0; p < data->meshes[i].primitives_count; p++)
```

Comment:
Regarding this TODO on line 5121: 
      
    // TODO: It seems LOAD_ATTRIBUTE() macro does not work as expected in some cases,

I didn't see any issues with my testing, so I'm not sure what issue this TODO is referencing or if it's out of date.

Reply:
I think this line could be removed